### PR TITLE
Minmaxscaler

### PR DESCRIPTION
**Pull request by prengsen**
Cambiamos el scaler por MinMaxScaler y creamos modelo de regresión logística, 

El accuracy_score fue de **0.8400**

### CODE REVIEW

**Comment by prengsen** [comment]
Mejoremos el accuracy, probemos agregar Hyper-parámetros a la regresión logística.

Consultar: https://scikit-learn.org/1.5/modules/generated/sklearn.linear_model.LogisticRegression.html
